.github/workflows/applicationsvalidation.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         with:
           path: cbsapplications
           repository: vitruv-tools/Vitruv-Applications-ComponentBasedSystems
-          ref: master
+          ref: main
           fetch-depth: 0
       - name: Checkout Matching Domains/Applications Branches
```

Comment:
Sorry for the unrelated comment, but I just saw that we have some outdated naming here (`Domains` would have to be replaced by `Framework/DSLs`). May we make that small modification in this PR? I don't think it's worth an individual PR.